templates/iot/examples/infinispan/manual/050-StatefulSet.yaml

```diff
@@ -91,9 +91,23 @@ spec:
         volumeMounts:
         - mountPath: /opt/jboss/infinispan-server/standalone/configuration/custom
           name: infinispan-configuration
+#        - mountPath: /opt/jboss/infinispan-server/standalone/data/devices
+# this would be cleaner but causes infinispan to crash because it cannot write in
+#   /opt/jboss/infinispan-server/standalone/data
+        - mountPath: /data/
+          name: infinispan-data
```

Comment:
As described, mounting the volume in `/opt/jboss/infinispan-server/standalone/data/devices` prevent infinispan to write in the `data` folder for some reason... 
Any ideas @lulf @ctron  ?

Reply:
In what way is it prevented? Is it a permissions issue on the PV you are testing with?